### DISCUSSION TITLE
Missing wWETH wrapped Ethereum

### DISCUSSION THREAD

**Discussion by harryjamesoates** [wormhole-foundation/wormhole#423] [Problems / bugs] [answered]
Hi,

I sent wWETH from my Sol Phantom wallet to Metamask through the wormhole and the transaction was an error due to insufficient funds to transfer. Both wallets noew show zero, is there a way to get the wWETH back?

Here is the transaction:
https://explorer.solana.com/tx/2fgeJtfNvjG571p7WA2ieQG6zfo5TY6ioKzPMoLTU8hNp1EDXFNrXG5WHgzNiNYYJGFV79YmKA8jjkmToh6Nv7Lj

Thanks!

**Comment by leoluk** [comment, marked as answer]
Wrong tx. Did you mean this one? https://explorer.solana.com/tx/5FrbU9ty7M5zJhMf8uYt959xGch1phcjneSdf6FSiBdkoTQjL5MSaR6u1C9jye71VGAV5ng7Fqag5gAg1BbsSCy9

The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected, and select both tokens from the picker.

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest (for reference only): 79b6b68bc3ff4c16815709b971816b5d8b28c621430a24078d62912e8fd055d9

Signed VAA bytes to submit to the contract:

> 0x01000000010d00d554b7b65e3d3d61d88f06596b36b17742389bf503f42a88e87c77063216014244d15691f93c87bdd62eb856e1e10536e068de246afe5c78971db4a399f3a4be01013bebe571a4604f2ed980ffb0d59a41e82206221c634651058f37044f0d76f9310a7f666f13ce6aace3242b351ce2638e08c100ef616cee242aee54e67fd63860010213950745f205523eabec1cebe0afc0b7d7e1b44db3ac2640fbbb9d842174214b06b573223346d94f441c243e61e129506d2eef3cfb1f87d47f4c40bdce57dd5500044dd70675c76d894078a35bfc5f443b37a9d76efa1b7be9be44565f1ffb3c5dfd7c381f7ae3c6661b3cd61cb9e1249788e579788792f0eb86624e7c7563feadfa0105cf6b0e06ab2b5d36a1bb725a37a51f3a14b6eab3e8b562c9323a57bfb41a520a612b9a843b67fe390567932ff96b6e3dc7a59fe114da32db267db9ac0ff7e5300006362d13c93acde13b6c4bbf41a62319965412d6a823a05be3cabe960605b7ecfd0bd083675e078c0fe7ac765cad33cc2622185ee0cfd699f0a7eb6ae5565234270007138c875f5bb61085d14fea643ddc72cf55c3006750bf63931ac554ea4a0f983d087dac4e6099689edb4f83b99c69041d1f2df765b6d620aab1e8822e6fd66105010886d97f2176928c71d56d37ab0709ac38364b4d142006494608558c521391d9426c91982f70fe96156c61d918238ff1ae9e0bc1b17b85c661e6e8142feef8d1cd01097a49cf36a5c993a46c414d2bd5e6e4fde11be3f400132c96e94eb0a2ded29e1d533bb7e21832df3d16abeb8ec5c79239ca78fb620b9f1dd1b4fd71c0b36cf13f000a70b63d04e236eed1d4288b29c04f9b5f134cf916f092ab1219d7fbc6212c827f43cc48af52f0a1349a89b79b22b450a62a59a3c15479ae7b863f4af9ca280356010df8b1b8d567683467e3a5d7d9cdf3d95213004638294aa7d8d44cef4cd802820f49d52a08d47e4aba5929873538f8cfb4b7f1da0bd6fe6c0fa2d46cad9dbdb9ff0111f6fa275c3d33cdfe124b13f2fb6e529cc00e76711bc58be7ab8bf7b8322610857b2eee80def7b161301b825ca545e4d3357bc6c24c2984b03e87c5b82897e5fe0012bc932cf36f01f131212a79aef13d5d29b932f660d165ee13a6fbcb4b8bf2b6a53ccc05ebde4637783cdabe154d735a869a10cfc673fc6e9cff089d87539c15df00613dd3951000006526010277330f10c4699311fa9c7b9031357b58eef2befb4f2dedbab40e389bc60cfd9a000000000000000000000000313714a629bd649150b8ab6c394a69ed44e6adcc02000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20900000000000000000000000000000000000000000000000000000000059a5380
